Jenkinsfile

```diff
@@ -7,128 +7,285 @@ pipeline {
+											for filename in $(ls *.${extension})
+											do
+												mv ${filename} unsigned-${filename}
+												curl -f -o ${filename} -F file=@unsigned-${filename} https://cbi.eclipse.org/macos/codesign/sign
+												if [[ $? != 0 ]]; then
+													echo "Signing of ${filename} failed"
+													exit 1
+												else
+													rm unsigned-${filename}
+												fi
+											done
+										fi
+									}
+									
+									binaryFragmentsRoot=${WORKSPACE}/eclipse.platform.swt.binaries/bundles
+									
+									if [[ ${PLATFORM} == cocoa.macosx.* ]]; then
+										echo "Sign ${PLATFORM} libraries"
+										fn-sign-files jnilib https://cbi.eclipse.org/macos/codesign/sign
+		
+										cp *.jnilib ${binaryFragmentsRoot}/org.eclipse.swt.${PLATFORM}/
+									
+									elif [[ ${PLATFORM} == gtk.linux.* ]]; then
+										echo "Removing existing webkitextensions"
+										#TODO: can the webkit handling be removed?!
+										rm -r ${binaryFragmentsRoot}/org.eclipse.swt.${PLATFORM}/webkit*/
+										
+										cp *.so ${binaryFragmentsRoot}/org.eclipse.swt.${PLATFORM}/
+										cp -r webkitextensions* ${binaryFragmentsRoot}/org.eclipse.swt.${PLATFORM}/
+									
+									elif [[ ${PLATFORM} == win32.win32.* ]]; then
+										echo "Sign ${PLATFORM} libraries"
+										fn-sign-files dll https://cbi.eclipse.org/authenticode/sign
+										
+										cp *.dll ${binaryFragmentsRoot}/org.eclipse.swt.${PLATFORM}/
+									fi
+									
+									cd ${WORKSPACE}/eclipse.platform.swt.binaries
+									echo "git status before add"
+									git status
+									git add --all *
+									echo "git status after add"
+									git status
 								'''
 							}
 						}
 					}
 				}
 			}
+		}
+		stage('Build') {
+			steps {
+				wrap([$class: 'Xvnc', useXauthority: true]) {
+					withEnv(["JAVA_HOME=${ tool 'openjdk-jdk17-latest' }"]) { // TODO: is this necessary since it's defined as jdk tool?
+						dir ('eclipse.platform.swt.binaries') {
+							sh '''
+								mvn install \
+									--batch-mode -Pbuild-individual-bundles -DforceContextQualifier=zzz \
+									-Dcompare-version-with-baselines.skip=true -Dmaven.compiler.failOnWarning=true
+							'''
+						}
+						dir ('eclipse.platform.swt') {
+							sh '''
+								mvn clean verify \
+									--batch-mode -Pbuild-individual-bundles -DcheckAllWS=true -DforkCount=0 \
+									-Dcompare-version-with-baselines.skip=false -Dmaven.compiler.failOnWarning=true \
+									-Dmaven.test.failure.ignore=true -Dmaven.test.error.ignore=true
+							'''
+						}
+					}
+				}
+			}
 			post {
 				always {
 					junit 'eclipse.platform.swt/tests/*.test*/target/surefire-reports/*.xml'
 					archiveArtifacts artifacts: '**/*.log,**/*.html,**/target/*.jar,**/target/*.zip'
-					discoverGitReferenceBuild referenceJob: 'eclipse.platform.swt/master'
-					recordIssues publishAllIssues: true, tools: [java(), mavenConsole(), javaDoc()]
```

Comment:
This was added in #521, but the releng JIPPs seems to not have the corresponding plug-ins.
Eventually they should be installed so that can be used again.